src/components/container/container/container.stories.ts

```diff
@@ -0,0 +1,100 @@
+import type { InputType } from '@storybook/types';
+import type { ArgTypes, Args, Meta, StoryObj } from '@storybook/web-components';
+import { type TemplateResult, html } from 'lit';
+
+import { sbbSpread } from '../../core/dom';
+
+import '../../button';
+import '../../title';
+import './container';
+
+import readme from './readme.md?raw';
+
+const containerContent = (title: string, color: string): TemplateResult => html`
+  <div style="overflow: auto">
```

Comment:
<img width="1032" alt="image" src="https://github.com/lyne-design-system/lyne-components/assets/48486994/f4be57df-bd6f-410c-89eb-a64a15522ef4">

Otherwise when top element has a margin-top we get things like this